cmd/drainer/README.md

```diff
@@ -41,6 +41,8 @@ Usage of drainer:
       prometheus pushgateway address, leaves it empty will disable prometheus push
   -metrics-interval int
       prometheus client push interval in second, set "0" to disable prometheus push (default 15)
+  -node-id string
+      the ID of drainer node; if not specify, we will generate one from hostname and the listening port
```

Comment:
```suggestion
      the ID of drainer node; if not specified, we will generate one from hostname and the listening port
```

Reply:
Updated. The README of pump is also updated. @suzaku